Documentation/contributors/design-proposals/related-images.md

```diff
@@ -0,0 +1,192 @@
+# Related Images
+
+Status: Pending
+
+Version: Alpha
+
+Implementation Owner: ecordell 
+
+## Motivation
+
+Operators often need to make use of other container images to perform their functions as operators. 
+
+## Proposal
+
+Introduce a new field `relatedImages` to the `ClusterServiceVersion` spec. 
+
+### ClusterServiceVersion Spec Changes
+
+A new section `relatedImages` is added to the ClusterServiceVersionSpec.
+
+```yaml
+kind: ClusterServiceVersion 
+metadata:
+  name: etcd-operator
+spec:
+  relatedImages:
+  - name: default
+    image: quay.io/coreos/etcd@sha256:12345 
+    annotation: default
+  - name: etcd-2.1.5
+    image: quay.io/coreos/etcd@sha256:12345 
+  - name: etcd-3.1.1
+    image: quay.io/coreos/etcd@sha256:12345 
```

Comment:
WDYT about listing these as objects instead of string?  Something like:

```
- name: etcd-3.1.1
  image:
    registry: quay.io
    repository: coreos/etcd
    reference: sha256:12345
```

In one of my use cases, I'd like to rebuild operator metadata when automatically rebuilding one of the operands.  It would be nice to not have to *parse* the `image` string in order to substitute in a new sha256 value automatically.